baremaps-openapi/src/main/resources/openapi.yaml

```diff
@@ -921,6 +1070,58 @@ components:
           nullable: true
         link:
           $ref: '#/components/schemas/link'
+    tileSet:
+      type: object
+      required:
+        - tilejson
+        - tiles
+      properties:
+        tilejson:
+          type: string
+          pattern: "\\d+\\.\\d+\\.\\d+\\w?[\\w\\d]*"
```

Comment:
Same I'd add `format: uuid` instead of an esotheric regexp :-)

Reply:
Leaving this as it represents a semver string :-)